.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+name: build-images
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GHCR_IMAGE: ghcr.io/${{ github.repository }}
+  DOCKERHUB_IMAGE: ${{ github.repository }}
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          body: |
+            Directus ${{ github.ref }}
+          draft: false
+          prerelease: false
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+
+      - uses: c-hive/gha-npm-cache@v1
+      - run: npm ci
+      - run: npm run build
+      - run: node docker/pack
+
+      - name: Cache build artifacts
+        uses: actions/cache@v2
+        with:
+          path: '**/dist'
+          key: build-artifacts-${{ github.sha }}
+
+  publish-npm:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Check NPM_TOKEN secret
+        if: secrets.NPM_TOKEN == null || secrets.NPM_TOKEN == ''
```

Comment:
Turns out this is actually not a thing you can do :C 

https://github.com/actions/runner/issues/520

Reply:
o_O